README.md

```diff
@@ -41,14 +41,32 @@ $Env:Path += ";$(pwd)"
 
 ## Getting the Code and Building Electron
 
-After installing build-tools, you can run a new Electron build with this command:
+After installing build-tools, you'll need to **create a new directory** for the Electron build.
+In your terminal, you'll need to run something like:
+
+```sh
+# create directory
+mkdir electron-gn
+
+# get its path (on Unix):
+cd electron-gn && pwd
+
+# get its path (on Windows):
+cd electron-gn && cd
+```
+
+in the directory where you typically place your code projects.
+
+After you've created this new directory, you can run a new Electron build with this command:
```

Comment:
This part isn't necessary -- `e init` will try to create the directory if it doesn't already exist.

In the literal case of `/path/to/new/electron/directory` dying with an error -- that's happening because there's no permission to create `/path`. If the user had used `/tmp/path/to/new/electron/directory` instead, that would have succeeded.